v2/googlecloud-to-snowflake/Readme.md

```diff
@@ -0,0 +1,8 @@
+# GoogleCloud to Snowflake Dataflow Templates
+
+The Currently supported template under GoogleCloud to Snowflake template groups is a Streaming pipeline which ingests 
+data from Pub/Sub subscription and outputs the messages into Snowflake. 
+
+The list of readme.md documents for templates under GoogleCloud to Snowflake set is:
+
+1. PubSub to Snowflake (Readme.md)
```

Comment:
Can you add a link here to follow convention?
https://github.com/GoogleCloudPlatform/DataflowTemplates/blob/4df12abc2da59a1c6baee7cf79dd05a746f158a4/v2/pubsub-binary-to-bigquery/README.md